java/sample-apps/aws-sdk/README.md

```diff
@@ -24,6 +24,8 @@ terraform init
 terraform apply
 ```
 
+For the agent version, to optionally enable a pipeline to send Prometheus metrics to a Prometheus backend, provide the ARN of a lambda layer with a custom collector configuration in a file named `config.yaml` when prompted after running the `terraform apply` command. 
```

Comment:
Always capitalize Lambda
and use full form "AWS Lambda" in the first usage, then later can shorten just "Lambda"